### ISSUE TITLE
Setting to auto check for Pack ID mismatch errors

### ISSUE THREAD

**Issue by anandananda** [restic/restic#3684]
Output of `restic version`
--------------------------

 0.12.1

What should restic do differently? Which functionality do you think we should add?
----------------------------------------------------------------------------------

 Optional setting so Restic auto checks every new snapshot for Pack ID mismatch errors and header zero value errors. Like --read-data, but maybe runs more slowly with less resources and runs in the background. User is notified if errors are found.

What are you trying to do? What problem would this solve?
---------------------------------------------------------

 With this new feature the user won't be surprised to find that some/all of their dozens/hundreds of snapshots are fatally corrupted with Pack ID mismatch errors. They will get notified the first time this error happens. I think this will be especially valuable for newbies (like me) who don't realize they have to manually do a --read-data check on every new snapshot to make sure their backups are good.

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------

 Um, well, Restic will make me happier when I know that I can set it to do a low-resource --read-data check on all my new snapshots. That will bring happiness. Thanks for your consideration and all your hard work on improving Restic. Cheers! Advat at jaiamma @ gmail . com

**Comment by MichaelEischer** [comment]
Do you know whether some of the damaged packs were actually damaged right from the start? Typically incomplete files don't show up immediately but only after a certain time. So what you might be looking for is #3326. Would that solve your problem or did have something else in mind?

**Comment by MichaelEischer** [comment]
Closing as duplicate of https://github.com/restic/restic/issues/3326.
